ui_framework/src/components/table/_index.scss

```diff
@@ -0,0 +1,12 @@
+$kuiTableCellPadding: $kuiSize;
+$kuiTableFontSize: 14px;
+
+/**
+ * 1. Prevent cells from expanding based on content size. This substitutes for table-layout: fixed.
+ */
+@mixin kuiTableCell {
```

Comment:
Can you show an example in your docs how to set the width directly. Pretty common to individually set width to force things like icons or avatars to collapse. For example, I want to make the icon column as small as possible, how would you propose doing that?

![image](https://user-images.githubusercontent.com/324519/28842673-6e3445e6-76b3-11e7-9430-14eca7c65e8b.png)